src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -88,14 +88,13 @@ internal static StrongNameKeys Create(ImmutableArray<byte> publicKey, CommonMess
 
         internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider messageProvider)
         {
-            if (string.IsNullOrEmpty(keyFilePath))
+            if (string.IsNullOrEmpty(keyFilePath) || !PathUtilities.IsAbsolute(keyFilePath))
```

Comment:
I am not sure whether changing this condition is a right thing to do. Can we keep the assert instead?